osg-token-renewer.py

```diff
@@ -133,10 +152,12 @@ def main():
     if not cfgx or not validate_config_dict(cfgx):
         sys.exit(1)
 
-    add_all_accounts(cfgx)
-    make_all_tokens(cfgx)
+    errors = add_all_accounts(cfgx)
+    errors += make_all_tokens(cfgx)
+
+    return errors
 
 
 if __name__ == '__main__':
-    main()
+    sys.exit(main() > 1)
```

Comment:
```suggestion
    sys.exit(main() > 0)
```

You initiate `errors =0`

Reply:
Wow, fumble!  Good catch.